include/tvm/runtime/metadata.h

```diff
@@ -75,6 +75,13 @@ struct TVMMetadata {
   const struct TVMTensorInfo* outputs;
   /*! \brief Number of elements in `outputs` array. */
   int64_t num_outputs;
+  /*! \brief Memory Pools needed by the AOT run_model function.
```

Comment:
You are not using `run_model` anymore, are you? In your previous branch indeed I'm seeing `tvmgen_default_run_model` generated, but after rebase it is replaced with `tvmgen_default___tvm_main__`.

Reply:
That is right!.

We had run (entry_point), run_model and  __tvm_main__.
The entry_point is supposed to call run_model, however, run_model is identical to tvm_main -- therefore, it was removed due to no need of maintaining two symbols for relay and tir versions of main.

So I think it needs to be tvm_main now.